contracts/Core/StakeManager.sol

```diff
@@ -20,7 +20,18 @@ contract StakeManager is Initializable, ACL, StakeStorage {
     IRewardManager public rewardManager;
     RAZOR public razor;
     IVoteManager public voteManager;
-
+    //[math.floor(getMaturity2(i*10000)) for i in range(1,100)]
+    uint256[] public maturities =
```

Comment:
How are we using maturities in the new mechanism? And any reason to select these numbers?

Reply:
Current we are using sqrt(). Since the number of input values are limited, we are using a lookup table instead of calculating sqrt, which will require external library.